### PR TITLE
sysfs: implement SenseContinuous in ThermalSensor

### PR DESCRIPTION
**Pull request by davidsansome**

### CODE REVIEW

**Comment by maruel** [comment]
The code lgtm but please fix the test.

**Comment by davidsansome** [comment]
I fixed the test again :D

File: cmd/thermal/main.go [hunk 1]
```diff
@@ -12,13 +12,16 @@ import (
...
 	"os"
+	"time"
 
 	"periph.io/x/periph/conn/physic"
 	"periph.io/x/periph/host"
 	"periph.io/x/periph/host/sysfs"
 )
 
 func mainImpl() error {
+	cont := flag.String("cont", "", "Reads continuously from the sensor with this name")
```
Comment (maruel) [inline cmd/thermal/main.go]: I think it should be `-dev` and looks as:
- If not specified, read all devices found
- If specified, read only this device

If you want to simplify your life, you could require `-dev` when using `-interval`.

**Comment by maruel** [comment]
gohci

File: cmd/thermal/main.go (hunk 1 above)
Comment (davidsansome) [reply cmd/thermal/main.go]: Done, sorry for the delay again!

**Comment by maruel** [comment]
gohci

**Pull request merged**